covenant/covenant_test.go

```diff
@@ -254,7 +255,7 @@ func FuzzAddCovenantSig(f *testing.F) {
 
 		// check the sigs are expected
 		expectedTxHash := testutil.GenRandomHexStr(r, 32)
-		mockClientController.EXPECT().SubmitCovenantSigs(covSigsSet).
+		mockClientController.EXPECT().SubmitCovenantSigs(gomock.Any()).
```

Comment:
it calls the same covenant sigs, but the mock considers it a different value :thinking: 
ex.:

```shell
         expected call at /home/rafilx/projects/github.com/babylonlabs-io/covenant-emulator/covenant/covenant_test.go:257 doesn't match the argument at index 0.
            Got: [0xc00152fc20 0xc0006f12c0] ([]*types.CovenantSigs)
            Want: is equal to [0xc0015d1ec0 0xc000f69c20] ([]*types.CovenantSigs)
```

Reply:
I thought this might be due to pointer address but how it was passing before

Reply:
Any idea here?

Reply:
hmm I think now adaptor signatures add random aux data every time the adaptor sig is made, so those objects won't be equal

Reply:
yep confirmed it with @SebastianElvis , now EncSig add random aux data to encryptor so we can't make this comparision. Lets leave it as `gomock.Any()`